src/test/run-pass-fulldeps/qquote.rs

```diff
@@ -9,59 +9,56 @@
 // except according to those terms.
 
 // ignore-cross-compile
-// ignore-pretty
 
 #![feature(quote, rustc_private)]
 
 extern crate syntax;
 
-use syntax::ast;
-use syntax::codemap;
-use syntax::parse;
-use syntax::print::pprust;
+use syntax::codemap::DUMMY_SP;
+use syntax::print::pprust::*;
 
-trait FakeExtCtxt {
```

Comment:
Right, so I agree with that part, but if you're going to change that, there are 2 other places where a `FakeExtCtxt` is made, and it would be good to fix them as well. As for this file's `main` function, there's really no need to change it; it's just noise :(